src/Drawer/Drawer.js

```diff
@@ -197,8 +199,16 @@ class Drawer extends Component {
     }
   };
 
+  getTranslatedWidth() {
+    if (this.props.width <= 1) {
+      return this.props.width * window.innerWidth;
```

Comment:
The difference with the `<Popover />` is that `window.innerWidth` is not called on the server. Here, it's called in the init render.
What about using a large value like `10000` instead of `null`? A comment would be good too. We need to explain that we do our best on the server to render a consitent UI.